### PR TITLE
WGs housecleaning

### PR DESCRIPTION
**Pull request by jlewi**
* Move the wgs files out of the proposals directory
* Create an example directory wg-foo to serve as an example.

### CODE REVIEW

File: wgs/wg-lifecycle.md [hunk 1]
```diff
@@ -10,17 +10,23 @@ Out of scope for this document: [subproject] creation.
...
-- [ ] Submit a PR that will add rows to [wgs.yaml] using the [generator doc]; this will create README files and OWNERS_ALIASES files for your new directory in `kubernetes/community`
+- [ ] Submit a PR that:
+      * adds rows to [wgs.yaml]
+      * add rows to [owner_aliases.yaml]
+        * See https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners_aliases
+      * Create directory `wgs/wg-<YOUR WG>` with 
+           * README.md
+             * See [kubeflow/community#372](https://github.com/kubeflow/community/issues/372)
+           * charter.md
+           * [OWNERS](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md)
```
Comment (andreyvelich) [inline wgs/wg-lifecycle.md:28]: List doesn't work for me in display mode here.

Maybe it has wrong indentation ?
```suggestion
- [ ] Submit a PR that:
  * adds rows to [wgs.yaml]
  * add rows to [owner_aliases.yaml]
    * See https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners_aliases
  * Create directory `wgs/wg-<YOUR WG>` with 
    * README.md
      * See [kubeflow/community#372](https://github.com/kubeflow/community/issues/372)
    * charter.md
    * [OWNERS](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md)
```

Comment (jlewi) [reply wgs/wg-lifecycle.md]: Thanks @andreyvelich ; FIxed PTAL.

**Pull request merged**